README.tfdoc.hcl

```diff
@@ -78,7 +78,7 @@ section {
       ```hcl
       module "terraform-aws-cognito-user-pool" {
         source  = "mineiros-io/cognito-user-pool/aws"
-        version = "~> 0.9.0"
+        version = "~> 0.9.2"
```

Comment:
Since we're using semantic versioning here, we can leave that value to `~> 0.9.0` which effectively means `0.9.0 >= 0.10.0`

Reply:
Ah I see, that makes sense